src/file_organizer/services/deduplication/embedder.py

```
@@ -125,8 +152,8 @@ def fit_transform(self, documents: list[str]) -> np.ndarray:
 
             return np.asarray(dense_embeddings)
 
-        except ValueError as e:
-            logger.error(f"Error during fit_transform: {e}")
+        except ValueError:
+            logger.error("Error during fit_transform", exc_info=True)
```

Ruff (0.15.9)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)